x-pack/filebeat/input/entityanalytics/provider/okta/internal/okta/okta.go

```diff
@@ -39,49 +39,12 @@ type User struct {
 	PasswordChanged       *time.Time     `json:"passwordChanged,omitempty"`
 	Type                  map[string]any `json:"type"`
 	TransitioningToStatus *string        `json:"transitioningToStatus,omitempty"`
-	Profile               Profile        `json:"profile"`
+	Profile               map[string]any `json:"profile"`
 	Credentials           *Credentials   `json:"credentials,omitempty"`
```

Comment:
Can you also add the [recovery_question.question](https://developer.okta.com/docs/reference/api/users/#credentials-object) to the credentials object. That is field also part of a feature request.

Reply:
I'll add it, but I think this is not a safe change. I would like to get input from infosec on their views on its inclusion given that knowledge of the question will give significant information to an attacker that has details about a victim.

Reply:
I'm checking with the user to understand what their use case is for having this information is. If they actually do want it and understand the risk, then maybe this should be opt-in in some manner.